src/validation/validationRules.js

```diff
@@ -18,6 +18,17 @@ export const signupInputRules = [
   check('password').exists().matches(/(?=.*\d)(?=.*[A-Z])(?=.*[a-z])(?=.*\W).{8,30}/).withMessage('At least 8 characters include symbols, uppercase, lowercase and number'),
 ];
 
+export const loginRules = [
+  check('email').trim().exists()
+    .withMessage('The email is required')
+    .isLength({ min: 1, max: 200 })
+    .withMessage('email can not be empty'),
+  check('password').exists()
+    .withMessage('The password is required')
+    .isLength({ min: 1, max: 200 })
```

Comment:
refer to signup validation rules. password minimum length should be 8, not 1 and also check the max limit.

Reply:
It is done